doc/api/stream.md

```diff
@@ -432,6 +432,18 @@ file.end('world!');
 // Writing more now is not allowed!
 ```
 
+[`stream.end()`][stream-end] will error with (in order of precedence):
```

Comment:
Can you clarify if it will throw or emit an error?

Reply:
How about now?